tools/bcdb_mock/models/schema/workloads/tfgrid_workloads_reservation_container_1.toml

```diff
@@ -33,6 +33,7 @@ mountpoint = (S)
 @url = tfgrid.workloads.reservation.network.connection.1
 network_id = (S)
 ipaddress = (ipaddress)
+public_ip6 = (B)
```

Comment:
We're also missing the capacity type here:
```
@url = tfgrid.workloads.reservation.container.capacity.1
# Number of vCPU
cpu = (I) 
# memory in MiB
memory = (I)
```

Reply:
where is that coming from?

Reply:
From here https://github.com/threefoldtech/zos/blob/master/pkg/provision/container.go#L68